### PR TITLE
[Docs] big cleanup

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: docs/reference/introduction.rst [hunk 1]
```diff
@@ -26,7 +26,8 @@ Comments
 --------
 
 Comments can be enabled or disabled depending on the policy selected while creating a Post.
-An email is sent every time a comment is added to a Post. The email contains the comment informations such as:
+An email is sent every time a comment is added to a Post.
+The email contains the comment informations such as:
```
Comment (greg0ire) [inline docs/reference/introduction.rst]: "information" is uncountable in english
```suggestion
The email contains the comment information such as:
```

File: docs/reference/installation.rst [hunk 2]
```diff
@@ -1,63 +1,62 @@
...
+----------------------
 
 .. code-block:: bash
 
-    composer require sonata-project/news-bundle sonata-project/doctrine-orm-admin-bundle
+    composer require sonata-project/news-bundle
 
+.. code-block:: bash
+
+     sonata-project/doctrine-orm-admin-bundle
```
Comment (kunicmarko20) [inline docs/reference/installation.rst]: missing composer require?

File: docs/reference/installation.rst [hunk 3]
```diff
@@ -83,29 +86,23 @@ If you want to use the API, you also need ``friendsofsymfony/rest-bundle`` and `
                         #ApplicationSonataNewsBundle: ~
                         SonataNewsBundle: ~
 
-
-* Import the ``sonata_news.yml`` file and enable json type for doctrine:
+Register JsonType
```
Comment (kunicmarko20) [inline docs/reference/installation.rst]: is this still needed?

Comment (greg0ire) [reply docs/reference/installation.rst]: Should be replace with a constraint on Doctrine. And we should deprecate this in favor of `vendor/doctrine/dbal/lib/Doctrine/DBAL/Types/JsonArrayType.php`

File: docs/reference/advanced_configuration.rst [hunk 4]
```diff
@@ -48,6 +43,20 @@ Advanced Configuration
                 controller:  SonataAdminBundle:CRUD
                 translation: SonataNewsBundle
 
+.. code-block:: yaml
+
+    # config/packages/sonata_classification.yaml
+
+    sonata_classification:
+        class:
+            collection: Application\Sonata\ClassificationBundle\Entity\Collection
```
Comment (core23) [inline docs/reference/advanced_configuration.rst]: Please use the new namespace

Comment (OskarStark) [reply docs/reference/advanced_configuration.rst]: Done in a separate commit

**Pull request merged**